documentation/assemblies/oauth/assembly-oauth-authentication.adoc

```diff
@@ -5,15 +5,19 @@
 [id='assembly-oauth-authentication_{context}']
 = Using OAuth 2.0 token-based authentication
 
-Strimzi supports the use of OAuth 2.0 authentication using the _SASL OAUTHBEARER_ mechanism.
+Strimzi supports the use of OAuth 2.0 authentication using the _SASL_OAUTHBEARER_ and _SASL_PLAIN_ mechanisms.
 
 OAuth 2.0 enables standardized token-based authentication and authorization between applications, using a central authorization server to issue tokens that grant limited access to resources.
 
+Kafka brokers and clients both need to be configured to use OAuth 2.0. 
 You can configure OAuth 2.0 authentication, then xref:assembly-oauth-authorization_{context}[OAuth 2.0 authorization].
 
-OAuth 2.0 authentication can also be used in conjunction with `simple` or OPA-based xref:con-securing-kafka-authorization-str[Kafka authorization].
+[NOTE]
+====
+OAuth 2.0 authentication can also be used in conjunction with `simple` or Open Policy Agent (OPA) based xref:con-securing-kafka-authorization-str[Kafka authorization].
```

Comment:
It is a bit weird to use only the `type` name for one and full name for the other. Would this be better?

```suggestion
OAuth 2.0 authentication can also be used in conjunction with Simple or Open Policy Agent (OPA) based xref:con-securing-kafka-authorization-str[Kafka authorization].
```